frontend/src/components/user/DialogChangeMail.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <dialog-form
+    v-model="showDialog"
+    :title="$tc('components.user.dialogChangeMail.title')"
+    :submit-action="(status == 'initial') ? sendChangeMailRequest : null"
+    :cancel-action="(status == 'initial') ? null : close"
+    :cancel-label="'Close'"
+    submit-color="success"
+    max-width="600px">
+    <template #activator="scope">
+      <slot name="activator" v-bind="scope" />
+    </template>
+    <div v-if="status == 'initial'">
+      <e-text-field
+        v-model="entityData.newEmail"
+        :name="$tc('entity.user.fields.email')"
+        autofocus
+        vee-rules="required" />
+      <p class="mt-5">
+        {{ $tc('components.user.dialogChangeMail.message') }}
+      </p>
+    </div>
+    <div v-if="status == 'success'">
+      {{ $tc('components.user.dialogChangeMail.success') }}
```

Comment:
I would add a hint here that you have to verify the new email address, before you can use it